src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -753,6 +754,14 @@ export class PatientViewPageStore {
         }
     }, undefined);
 
+    getGeneFilterDefault(config:IServerConfig):GeneFilterOption {
+        const propName = 'skin_patientview_filter_genes_profiled_all_samples';
```

Comment:
my gut says this should be a util since it doesn't use `this`

Reply:
Yes I agree with your gut feeling. Method was moved to new utils file. Also moved unit tests to new file.